pycoal/tests/test.py

```diff
@@ -26,7 +26,7 @@ def _remove_files(listOfFileNames):
             pass
 
 # file names of USGS Digital Spectral Library 06 in ENVI format
-libraryFilenames = ["s06av95a_envi.hdr", "s06av95a_envi.sli"]
+libraryFilenames = ["s07_AV95_envi.hdr", "s07_AV95_envi.sli"]
 
 # set up test module before running tests
 def setup_module(module):
```

Comment:
You are not using any of the test files you've introduced. You either need to use them or remove them.